lua/gitblame/init.lua

```diff
@@ -621,61 +645,79 @@ local function clear_all_extmarks()
     end
 end
 
--- this function is uesed to verify the config info for debounce
---- @return boolean
-local verify_debounce_info = function()
-    if vim.g.gitblame_schedule_event ~= "CursorMoved" and vim.g.gitblame_schedule_event ~= "CursorHold" then
-        vim.notify(
-            string.format("event is error: %s, it just can be CursorMoved or CursorHold", vim.g.gitblame_schedule_event),
-            vim.log.levels.ERROR,
-            {}
-        )
-        return false
-    end
-    if vim.g.gitblame_clear_event ~= "CursorMovedI" and vim.g.gitblame_clear_event ~= "CursorHoldI" then
-        vim.notify(
-            string.format("event is error: %s, it just can be CursorMoved or CursorHold", vim.g.gitblame_clear_event),
-            vim.log.levels.ERROR,
-            {}
+-- Validates the `parameter_name` against `available_values`. Returns an error message
+-- if `parameter_name` doesn't match `available_values`.
+--
+---@param parameter_name string
+---@param available_values object[]
```

Comment:
Nitpick: It's actually a string table

```suggestion
---@param available_values string[]
```

Reply:
It should work for non-strings, as well :)